trackers/huggingface/graphs.py

```diff
@@ -613,3 +626,165 @@ def results_table(df: pd.DataFrame):
         df = df[[model_name in x for x in df["Model Name"]]]
 
     st.dataframe(df, height=min((len(df) + 1) * 35, 35 * 21))
+
+
+def device_funnel(df: pd.DataFrame) -> None:
+    """
+    Show count of how many models compile, assemble, etc
+    """
+    summ = DeviceStageCount(df)
+
+    stages = [
+        "All models",
+        "Export to ONNX",
+        "Optimize ONNX file",
+        "Convert to FP16",
+        "Acquire Performance",
+    ]
+    cols = st.columns(len(stages))
+
+    for idx, stage in enumerate(stages):
+        with cols[idx]:
+            st.markdown(stage)
+
+    # Show Sankey graph with percentages
+    sk_val = {
+        "All models": f"{summ.all_models} models - 100%",
+        "Convert to ONNX": f"{summ.base_onnx} models - "
```

Comment:
```suggestion
        "Converts to ONNX": f"{summ.base_onnx} models - "
```

match the style below?

Reply:
Matching style!